bson/src/main/org/bson/codecs/configuration/CodecRegistries.java

```diff
@@ -17,6 +17,7 @@
 package org.bson.codecs.configuration;
 
 import org.bson.codecs.Codec;
+import org.bson.internal.ProvidersCodecRegistry;
```

Comment:
nit: Extra import

Reply:
It seems like it's needed, as we new one of these up on line 89